### DISCUSSION TITLE
How to follow the customer journey of the users ?

### DISCUSSION THREAD

**Discussion by placko** [umami-software/umami#1767] [Q&A]
Hello,
Is there a way to implement a goal manager to follow the customer journey of the users (Event Management).
Sincerely
Nicolas

**Comment by briancao** [comment]
Not at this time. Those features are planned for Q2 this year.

**Comment by chreniuc** [comment]
@briancao does this mean that I will be amble to do something like this:

```html
<script async defer data-website-id="123etc
   src="https://umami-colect/collect.js"
   username="John"></script>
```
And in the dashboad to be able to filter the visits from John? I would be interested in something like this.

**Reply by briancao** [reply]
We've yet to decide exactly how granular funnels will be.  Furthermore, given how Umami anonymizes data, I doubt we'd provide the ability to track specific users like that.

But if you want to track specific users by manually inserting event_data, and then filter/query the funnel with that data, I suppose that could be a potential method.  We'll see in q2.
